control/controlclient/map.go

```diff
@@ -259,10 +262,39 @@ func (ms *mapSession) updateStateFromResponse(resp *tailcfg.MapResponse) {
 		ms.lastDERPMap = dm
 	}
 
+	var packetFilterChanged bool
+	// Older way, one big blog:
 	if pf := resp.PacketFilter; pf != nil {
+		packetFilterChanged = true
+		mak.Set(&ms.namedPacketFilters, "base", views.SliceOf(pf))
+	}
+	// Newer way, named chunks:
+	if m := resp.PacketFilters; m != nil {
```

Comment:
Is it safe to assume the name `"base"` will only be used by the full-monty filters from the above case?

Reply:
should say that "base" is reserved in comments?

Reply:
Up to the server. But probably.

I fleshed out the docs a bit.